benchmark_runner/common/elasticsearch/elasticsearch_operations.py

```diff
@@ -28,9 +28,19 @@ def __init__(self, es_host: str, es_port: str, es_user: str = '', es_password: s
         self.__es_user = es_user
         self.__es_password = es_password
         self.__timeout = timeout
+
+        import ssl
+        from elasticsearch.connection import create_ssl_context
+
+        ssl_context = create_ssl_context()
+        ssl_context.check_hostname = False
+        ssl_context.verify_mode = ssl.CERT_NONE
+        import urllib3
+        urllib3.disable_warnings()
+
         if self.__es_password:
-            self.__es_url = f"http://{self.__es_user}:{self.__es_password}@{self.__es_host}:{self.__es_port}"
-            self.__es = Elasticsearch([self.__es_url])
+            self.__es_url = f"https://{self.__es_user}:{self.__es_password}@{self.__es_host}:{self.__es_port}"
+            self.__es = Elasticsearch([self.__es_url], ssl_context=ssl_context)
```

Comment:
Need to keep password option seperatly and add more option for https in environment variable:
https://github.com/redhat-performance/benchmark-runner/blob/main/benchmark_runner/main/environment_variables.py#L39